definitions/divorce/json/CaseEventToFields/CaseEventToFields-general-email-nonprod.json

```diff
@@ -20,7 +20,7 @@
     "PageFieldDisplayOrder": 2,
     "DisplayContext": "MANDATORY",
     "PageID": "creategeneralemail",
-    "PageLabel": "General order",
+    "PageLabel": "Create general email",
```

Comment:
they are not used anyway - only title from the first field on page is used, but they were misleading.